### ISSUE TITLE
CMF评估问题

### ISSUE THREAD

**Issue by huaibovip** [Runshi-Zhang/UTSRMorph#9]
在CMF评估中，模型输入为 `x_in = torch.cat((x, y), dim=1)`，那么将产生一个形变场 ${\phi}_{x\rightarrow y}$ 。

正确的做法应该是使用将 `mov_point` 变换到 `y` 所在的目标空间，而不是 `fix_point`。

https://github.com/Runshi-Zhang/UTSRMorph/blob/1298c0845c09a21f0e5836ebe0a0c9152681d27c/submit_utsrmorph_CMF.py#L69-L78

https://github.com/Runshi-Zhang/UTSRMorph/blob/1298c0845c09a21f0e5836ebe0a0c9152681d27c/submit_utsrmorph_CMF.py#L96-L100

**Comment by Runshi-Zhang** [comment]
点位置的运算与图像的运算是反着的。图像运算moving到fixed，x+displacement=y。点是反着的从fixed到moving，y+displacement=x。

**Comment by huaibovip** [comment]
好的谢谢

**Comment by huaibovip** [comment]
能分享 `Abdomen MR-CT` 的预处理代码吗？十分感谢！！

**Comment by Runshi-Zhang** [comment]
上传了仿射变换代码，affine_registration.py

**Comment by huaibovip** [comment]
感谢您的回复
